express-post-json/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const app = express();
+app.listen(3000, () => console.log('Server is indeed listening'));
+
+let nextId = 1;
+
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const X = [];
```

Comment:
You really should avoid using `x` for a variable name, this will eventually come back to haunt you.